lib/controllers/skill-infrastructure-controller/index.js

```diff
@@ -168,6 +172,17 @@ module.exports = class SkillInfrastructureController {
         });
     }
 
+    /**
+     * Updates event endpoint if reuseEndpointForEvents is true on user config
+     * @param {object} userConfig user config
+     * @param {String} uri events endpoint
+     */
+    _updateEventsEndpoint(userConfig, uri) {
```

Comment:
current tests do not mock ask-resources file. extracted to private method to make a little easier to test.